pkg/bff/api/v1/api.go

```diff
@@ -19,6 +19,7 @@ type BFFClient interface {
 
 	// User Management Endpoints
 	Login(context.Context) error
+	ListUserRoles(context.Context) (map[string]struct{}, error)
```

Comment:
Why a `map[string]struct{}` instead of `[]string`? How does this render in JSON - something like below?

```json
{
    "Organization Leader": {},
    "Organization Contributor": {}
}
```

Reply:
I didn't want to impose ordering on it but I can make it a `[]string` if that's easier.

Reply:
I think `[]string` is probably simpler.